samples/corda/corda-simple-application/scripts/getAssetStatus.sh

```diff
@@ -8,3 +8,15 @@ CORDA_PORT=10009 ./clients/build/install/clients/bin/clients get-asset-states-by
 echo -e "\n--\n"
 echo "PartyC Tokens:"
 CORDA_PORT=10012 ./clients/build/install/clients/bin/clients get-asset-states-by-type t1
+
+if [ "$1" = "2" ]; then
```

Comment:
Does `2` imply the second Corda network?

Reply:
Yes, if we don't pass any parameter, it will only report for 1st network.